src/systems/ui-hotkeys.js

```diff
@@ -48,6 +48,12 @@ AFRAME.registerSystem("ui-hotkeys", {
     if (this.userinput.get(paths.actions.toggleCamera)) {
       this.el.emit("action_toggle_camera");
     }
+
+    if (this.userinput.get(paths.actions.toggleUI)) {
+      if (this.el.sceneEl.is("entered")) {
```

Comment:
May be useful to hide the UI from the lobby, too

Reply:
Yeah, I had considered that originally, but it felt a bit weird since that UI is more of a 'landing page' than "HUD" chrome

Reply:
On the Spanish keyboard, what would the key be?

Reply:
it's wired right now to the backtick character. colloquially this is usually referred to as the tilde key in US